### ISSUE TITLE
How to use BatchNorm3D in inplace_abn and inplace_abn_sync?

### ISSUE THREAD

**Issue by JackieQiang** [mapillary/inplace_abn#39]
Inplace_abn is an amazing research! But this code seems only apply to 2D image.I want to use inplace_abn in 3d network with BatchNorm3D.I don't know how to change it.
Could you tell me how to solve this problem. very thankful!

**Comment by rotabulo** [comment]
@JackieQiang you could reshape the 5D tensor as a 4D tensor, apply 2d Inplace ABN and reshape the result back as a 5D tensor.

**Comment by rotabulo** [comment]
@JackieQiang have you actually tried? Our code should work with any number of dimensions.

**Comment by JackieQiang** [comment]
Thanks! It works. I previously felt inplace_abn_sync like pytorch BN which need use BatchNorm3D or BatchNorm2D.Inplace_abn_sync amazing. It only one method include any number of dimensions.

**Comment by rotabulo** [comment]
I close the issue.
